specter-desktop/umbrel-app.yml

```diff
@@ -44,15 +44,15 @@ path: ""
 defaultUsername: ""
 defaultPassword: ""
 releaseNotes: >-
```

Comment:
@aphex3k Would you mind adding some release notes from the intermediary version (v1.14.0) that had quite a few changes: https://github.com/cryptoadvance/specter-desktop/releases/tag/v1.14.0

Reply:
10-4